src/ServiceControl.LicenseManagement/LicenseManager.cs

```diff
@@ -1,11 +1,11 @@
-﻿namespace ServiceControlInstaller.Engine.LicenseMgmt
+﻿namespace ServiceControl.LicenseManagement
 {
     using System;
     using Particular.Licensing;
 
     public class LicenseManager
     {
-        public static DetectedLicense FindLicense()
+        public static DetectedLicense FindLicense(bool includeTrial = true)
```

Comment:
I think I added this, but I don't think this is necessary. It should be:

```suggestion
        public static DetectedLicense FindLicense()
```